server/src/main/java/io/crate/protocols/postgres/ClientMessages.java

```diff
@@ -114,6 +114,11 @@ static void sendFlush(ByteBuf buffer) {
         buffer.writeInt(4);
     }
 
+    static void sendSync(ByteBuf buffer) {
```

Comment:
Looks like it's used only in tests. Can we add VisibleForTesting or better move this to the test and remove here?